### PR TITLE
Add `RotationDesignable` support to `ib` #no-public-changes

### PR DESCRIPTION
**Pull request by tbaranes**
Related to #528

### CODE REVIEW

File: Sources/IB.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+//  IBAnimatableApp
+//
+//  Created by Tom Baranes on 17/01/2018.
+//  Copyright © 2018 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+
+private struct AssociatedKeys {
+  static var ib = "IBAnimatable"
```
Comment (JakeLin) [inline Sources/IB.swift]: can be `let`?

File: Sources/IB.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+public final class IB: NSObject, CornerDesignable {
```
Comment (JakeLin) [inline Sources/IB.swift]: I don't quite get it, why do we have two `IB` classes?

File: Sources/Others/IB.swift [hunk 3]
```diff
@@ -8,7 +8,8 @@
 
 import Foundation
 
-public final class IB: NSObject, CornerDesignable, FillDesignable, BorderDesignable {
+public final class IB: NSObject, CornerDesignable, FillDesignable, BorderDesignable,
```
Comment (JakeLin) [inline Sources/Others/IB.swift:11]: a space before `{` please 🤣

**Review by JakeLin — COMMENTED** [review]
Good progress, please have a look my comments

File: Sources/IB.swift (hunk 2 above)
Comment (tbaranes) [reply Sources/IB.swift]: Rebase failure 😔

File: Sources/IB.swift (hunk 1 above)
Comment (tbaranes) [reply Sources/IB.swift]: 👍

Comment (tbaranes) [reply Sources/IB.swift]: > Cannot pass immutable value as inout argument: 'ib' is a 'let' constant

This is why :D

**Review by JakeLin — APPROVED** [review]
👍  That's a lot cleaner

**Pull request merged**